### ISSUE TITLE
sap_install_media_detect: `find` module does not support `contains` with `file_type: directory`

### ISSUE THREAD

**Issue by berndfinger** [sap-linuxlab/community.sap_install#388]
as per https://docs.ansible.com/ansible/latest/collections/ansible/builtin/find_module.html.
The related task is https://github.com/sap-linuxlab/community.sap_install/blob/60f5f40497d54c46b0bd3f79ef1fef4ada148119/roles/sap_install_media_detect/tasks/detect_sapanydb_sapase.yml#L78 .

**Comment by berndfinger** [comment]
A solution would be to replace the following lines in all affected `find` modules (for example):
```
contains: "LINUXX86_64"
```
by:
```
patterns: ".*LINUXX86_64.*"
use_regex: true
```

**Comment by berndfinger** [comment]
Solved by [07641a2](https://github.com/berndfinger/community.sap_install/commit/07641a25be9db562200b5e88a6c34f59382c4c73).
